web/syncPoolHandler_elements.go

```diff
@@ -155,7 +157,7 @@ func (p *handlerPool) getElement(uid string) (*poolElement, bool, error) {
 
 		element = &poolElement{
 			uid:     uid,
-			handler: NewSyncUserHandler(uid, db, nil),
```

Comment:
I guess this nil causes a default config to be created/picked up? Which implies that there are some defaults somewhere, and they're not what you actually intend to use?